zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/CamundaExporterIT.java

```diff
@@ -255,6 +257,28 @@ void shouldNotErrorIfOldExporterRestartsWhileNewExporterHasAlreadyStarted(
         .isTrue();
   }
 
+  void shouldExportRoleRecord(
+      final Callable<RoleEntity> getResponse, final Record<RoleRecordValue> record)
+      throws Exception {
+    // given
+    CONFIG.setCreateSchema(true);
+    final var exporter = startExporter();
+
+    // when
+    exporter.export(record);
+
+    // then
+    final var responseRoleEntity = getResponse.call();
+
+    assertThat(responseRoleEntity)
+        .describedAs("User entity is updated correctly from the user record")
+        .extracting(RoleEntity::getRoleKey, RoleEntity::getName, RoleEntity::getAssignMemberKeys)
+        .containsExactly(
+            record.getValue().getRoleKey(),
+            record.getValue().getName(),
+            record.getValue().getEntityKey());
```

Comment:
@Ben-Sheppard - I left this call to `getEntityKey()` in place as the `RoleRecordValue` is coming from Zeebe. I was unsure if that class needed changed to `assignedMemberKeys` as well

Reply:
NADB but I'd remove this line as we don't expect it to be relevant to the handler